assembl/static2/js/app/components/debate/thread/post.jsx

```diff
@@ -49,7 +49,7 @@ export class EmptyPost extends React.PureComponent {
   }
 
   componentDidMount() {
-    this.props.measureTreeHeight(400);
+    this.props.measureTreeHeight(480);
```

Comment:
Could you please make this delay constant with a comment as to why it is the number that it is?

Reply:
Yes sure. I'm not really sure to understand the reason of this value. I asked to @cedricmessiant via Slack.

Reply:
But for sure, it's not a delay. It's a height value I guess.

Reply:
ah no, don't modify this, it's not a height, it's a delay of 400ms!